docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -734,6 +734,19 @@ WAL retention time, in seconds, to be used for tables for which a CDC stream was
 
 Default: `14400` (4 hours)
 
+## Metric export flags
+
+##### --export_help_and_type_in_prometheus_metrics
+
+Master metrics are available in Prometheus format at
+`http://localhost:7000/prometheus_metrics`.  This flag controls whether
+#TYPE and #HELP information is included as part of the Prometheus
+metrics output by default.  This choice can be overridden on a
```

Comment:
Thanks for documenting this.
At first glance, it is not obvious that 'show_help' is the alternative to the setting the gflag. Should we make that more explicit?

Reply:
"can be overridden on a per-scrape basis" seems pretty explicit.  What would you suggest to make it more obvious for you?

Reply:
Active voice helps. And put it in a new paragraph. "To override this flag on a per-scrape basis, pass the URL parameter `show_help` with the value `true` to include or `false` to not include type and help information."

Reply:
What's a URL parameter BTW? Anything we can link to?